### PR TITLE
Add documents about controllers inherited from kubernetes.

### PR DESCRIPTION
**Pull request by RainbowMango**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR set up the documentation about the controller we inherited from Kubernetes.
And takes the `serviceaccount-token` controller as a start.

**Which issue(s) this PR fixes**:
Fixes #655 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
cc @mrlihanbo @GitHubxsy

**Comment by Garrybest** [comment]
/lgtm
By the way, this doc happens to help me know more about tokens controller. Thanks!

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Comment by RainbowMango** [comment]
Thanks, guys. Let's wait for @Tingtal for the grammar check.

File: docs/userguide/configure-controllers.md [hunk 1]
```diff
@@ -3,39 +3,44 @@
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
 - [Configure Controllers](#configure-controllers)
-  - [Controllers Overview](#controllers-overview)
-  - [Configure Controllers](#configure-controllers-1)
+  - [Karmada Controllers](#karmada-controllers)
+    - [Configure Karmada Controllers](#configure-karmada-controllers)
+  - [Kubernetes controllers](#kubernetes-controllers)
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:8]:
```suggestion
  - [Kubernetes Controllers](#kubernetes-controllers)
```

File: docs/userguide/configure-controllers.md [hunk 2]
```diff
@@ -3,39 +3,44 @@
...
+    - [Recommended controllers](#recommended-controllers)
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:9]:
```suggestion
    - [Recommended Controllers](#recommended-controllers)
```

File: docs/userguide/configure-controllers.md [hunk 3]
```diff
@@ -50,7 +55,44 @@ E.g. Disable some controllers:
...
+Users are recommended to deploy the `kube-controller-manager` along with Karmada components. And the installation
+methods list in [installation guide][2] would help you deploy it as well as Karmada components.
+
+### Recommended controllers
+
+Not all controllers in `kube-controller-manager` are necessary for Karmada, if you are deploying
+Karmada by other tools, you might have to configure the controllers by `--controllers` flag just like what we did in
+[example of kube-controller-manager deployment][3].
+
+Following controllers are tested and recommended by Karmada.
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:75]:
```suggestion
The following controllers are tested and recommended by Karmada.
```

File: docs/userguide/configure-controllers.md [hunk 4]
```diff
@@ -50,7 +55,44 @@ E.g. Disable some controllers:
...
+
+#### namespace
+TODO.
+
+#### garbagecollector
+TODO.
+
+#### serviceaccount-token
+
+The `serviceaccount-token` controller runs as part of `kube-controller-manager`, one of its responsibility is
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:85]:
```suggestion
The `serviceaccount-token` controller runs as part of `kube-controller-manager`.
```

File: docs/userguide/configure-controllers.md [hunk 5]
```diff
@@ -50,7 +55,44 @@ E.g. Disable some controllers:
...
+it watches `ServiceAccount` creation and creates a corresponding ServiceAccount token Secret to allow API access.
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:86]:
```suggestion
It watches `ServiceAccount` creation and creates a corresponding ServiceAccount token Secret to allow API access.
```

File: docs/userguide/configure-controllers.md [hunk 6]
```diff
@@ -50,7 +55,44 @@ E.g. Disable some controllers:
 Use `-foo` to disable the controller named `foo`.
 
 > Note: The default controller list might be changed in the following release. The controllers enabled by last release
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:57]:
```suggestion
> Note: The default controller list might be changed in the future releases. The controllers enabled in the last release
```

File: docs/userguide/configure-controllers.md [hunk 7]
```diff
@@ -50,7 +55,44 @@ E.g. Disable some controllers:
...
+In addition to the controllers that are maintained by the Karmada community, Karmada also requires some controllers from
+Kubernetes. These controllers run as part of `kube-controller-manager` and are maintained by the Kubernetes community.
+
+Users are recommended to deploy the `kube-controller-manager` along with Karmada components. And the installation
+methods list in [installation guide][2] would help you deploy it as well as Karmada components.
+
+### Recommended controllers
+
+Not all controllers in `kube-controller-manager` are necessary for Karmada, if you are deploying
+Karmada by other tools, you might have to configure the controllers by `--controllers` flag just like what we did in
```
Comment (Tingtal) [inline docs/userguide/configure-controllers.md:72]:
```suggestion
Karmada using other tools, you might have to configure the controllers by `--controllers` flag just like what we did in
```

**Review by Tingtal — CHANGES_REQUESTED** [review]

**Pull request merged**